glpi-official/services/web/files/opt/startup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Run cron service.
+cron
+
+# Run command previously defined in base php-apache Dockerfile.
+apache2-foreground
+
+if [[ -f "/var/www/glpi/config/config_db.php"]]; then
+    echo "GLPI installed already"
+else
+    echo "GLPI config missing. Installing GLPI..."
+    ./opt/install_glpi.sh
+    echo "Installation finished"
+fi
```

Comment:
Unless I am wrong, this will not work.

Indeed, a container stops as long as the startup script is exiting. This is why we use `apache2-foreground` here, as it is a foreground process that is not exiting (unless on failure). So to be sure that this `if/else` block is executed, you have to move it prior to `apache2-foreground` execution.